### PR TITLE
Merge branch AMOS-9-helloworld-for-frontend into origin/dev

### PR DESCRIPTION
**Pull request by PhlppKrmr**

### CODE REVIEW

File: code/frontend/src/main.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './app.css'
```
Comment (leandertolksdorf) [inline code/frontend/src/main.js:1]: I think in the next sprint we should switch to TypeScript.

**Review by leandertolksdorf — APPROVED** [review]
works for me.

File: code/frontend/src/main.js (hunk 1 above)
Comment (PhlppKrmr) [reply code/frontend/src/main.js:1]: On Sunday @rabbit-zero, @jenswaechtler and me spoke about whether we want to use JavaScript or TypeScript, we decided on JavaScript. If you want to we can talk about it tomorrow.

**Comment by PhlppKrmr** [comment]
Made the mistake of selecting main as merge goal, is now resolved. Sorry for intervening PR.

**Review by motschel123 — APPROVED** [review]
Working on my machine. 
Nice example rendering 👍

**Pull request merged**